tasks/purgeChannel.js

```diff
@@ -0,0 +1,151 @@
+
+const cron = require('cron')
+const log = require('../helpers/logger')
+const { getAllGuilds } = require('../helpers/guildData')
+const { Collection } = require('discord.js')
+
+const tasks = {}
+let _client
+
+const limit = 100 // 100 is the discord API limit SADFACE.JPG
+
+const twoWeeksInMilliSeconds = 14 * 24 * 60 * 60 * 1000
+
+const checkIndividual = (message, maxAge) => {
```

Comment:
not a big fan of these names as they indicate how they are used but not what they do.
Would prefer something more like 

shouldDeleteMessage
canDeleteInBulk
shouldDeleteIndividually